### PR TITLE
Preparing v0.4.0

### PR DESCRIPTION
**Pull request by oualib**
@afard can you please add the following package in the build:
 - 'scikit-learn>=0.23.0',
 - 'shap>=0.36.0',
 - 'pandas>=0.23.0'
 - 'geopandas>=0.8.0',
 - 'descartes>=1.0.0',
 - 'shapely>=1.6.0'
 - 'anytree>=2.5.0',
 - 'python-highcharts>=0.4.1'

When it will be pulled, it is needed to create a new Tag 'v0.4.0' please.

Changes include:
 - geospatial
 - to_sklearn function
 - more options
 - shap evaluation

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -13,4 +13,11 @@ deps =
...
-    vertica_python
\ No newline at end of file
+    vertica_python
+    geopandas
+    scikit-learn
+    shapely
+    descartes
+    anytree
+    python-highcharts
+    shap
```
Comment (afard) [inline tox.ini:23]: I wonder if `shap==0.36` resolves the failure on Python 3.6 test.

**Comment by afard** [comment]
Removing `shap` from tox.ini has made an improvement as the error message has been changed. Now, the installation of `pyproj`, which is required by `geopandas`, is failing.
As you can see (https://travis-ci.com/github/vertica/VerticaPy/jobs/461083172), it says: "proj executable not found. Please set the PROJ_DIR variable. For more information see: https://pyproj4.github.io/pyproj/stable/installation.html".

**Comment by oualib** [comment]
@afard, I deleted geopandas/shap from the ini file and now it works. Can you please review, merge and tag this version?

**Review by afard — APPROVED** [review]
Later, we should find a way to make test_vDF_to_geopandas work at least for Python 3.7 and 3.8.

**Pull request merged**